lib/kennel/models/record.rb

```diff
@@ -92,11 +93,11 @@ def resolve_linked_tracking_ids!(*)
       def add_tracking_id
         json = as_json
         if self.class.parse_tracking_id(json)
-          invalid! "remove \"-- Managed by kennel\" line it from #{self.class::TRACKING_FIELD} to copy a resource"
+          invalid! "remove \"-- #{MANAGED_BY_KENNEL}\" line it from #{self.class::TRACKING_FIELD} to copy a resource"
         end
         json[self.class::TRACKING_FIELD] =
           "#{json[self.class::TRACKING_FIELD]}\n" \
-          "-- Managed by kennel #{tracking_id} in #{project.class.file_location}, do not modify manually".lstrip
+          "-- #{MANAGED_BY_KENNEL} #{tracking_id} in #{project.class.file_location}, do not modify manually".lstrip
```

Comment:
good refactor overall ... not sure why we need the env setter, so comment at the const definition would be nice to explain why it's useful